src/middlewares/parameters.ts

```diff
@@ -45,14 +37,10 @@ export const isUserId = (request: UserRequest, response: Response, next: NextFun
 
 // Checks the user is the captain of the team. If not, it will return an error
 export const isInTeamId = async (request: UserRequest, response: Response, next: NextFunction): Promise<void> => {
-  const token = getToken(request);
-
-  if (token) {
-    const decoded = jwt.verify(token, jwtSecret()) as Token;
-    const { userId } = decoded;
-    const user = await fetchUser(userId);
+  const { user } = request;
+  if (user) {
     // Compare user's teamId and teamId of the request
-    if (user && request.params.teamId === user.teamId) {
+    if (request.params.teamId === user.teamId) {
```

Comment:
```suggestion
    if (user.teamId === request.params.teamId) {
```
Bon, à ce niveau, c'est du pinaillage ^^

Reply:
J'ai pensé pareil mais je me suis dit que c'était logique aussi de comparer le param avec le user ;)